.github/ISSUE_TEMPLATE/tracking-issue.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Tracking issue
+about: Template for a tracking issue
+title: ''
+labels: tracking-issue
+assignees: ''
+
+---
+
+# Summary
+
+One paragraph to explain the feature.
+
+# Motivations
+
+Why are we doing this? What use cases does it support? What is the expected outcome?
+
+# Explanation
+
+Explain the proposal like it was the final documentation of this proposal.
+
+- What is changing for end-users.
+- How it works.
+- What is breaking?
+- Examples.
+
+# Implementation
+
+Explain the technical specificities that will need to be known or done in order to implement this proposal.
+
```

Comment:
Could you add a "Documentation draft" header which must be a short documentatio that user will probably see in the documentation, a short explanation on how to use the feature?

Reply:
It is already here https://github.com/meilisearch/MeiliSearch/pull/785/files#diff-cdcc247fa1ab0b29514a3e0c4ad2248dR18-R20 
It is not enough?